### PR TITLE
Mention coala quickstart

### PR DESCRIPTION
**Pull request by sils**

### CODE REVIEW

**Review by Naereen — APPROVED** [review]
Sounds good, but not enough yet.
I guess it should be mentioned on the main README.md file, and on the documentation as well.

File: _posts/2000-01-03-details.md [hunk 1]
```diff
@@ -31,6 +31,8 @@ There's a few caveats and you'll have to have a working docker setup for this to
 
 ### Your configuration will be automatically saved to the `.coafile` in the current directory. Go to our [bear documentation](https://coala.io/languages) to see what bears are available. Bears can be installed as pip packages.
 
+### You can generate a config file to start with using [coala-quickstart](https://github.com/coala/coala-quickstart/blob/master/README.rst)
```
Comment (gaocegege) [inline _posts/2000-01-03-details.md]: maybe you forget the full stop at the end.

Comment (sils) [reply _posts/2000-01-03-details.md]: check :+1:

**Comment by razvanch** [comment]
ack 9a2a3b0

**Comment by sils** [comment]
@rultor merge

**Pull request merged**